Added dedicated page for Ibexa Cloud CLI (#3198)
ezsystems/developer-documentation: docs/ibexa_cloud/ibexa_cloud_cli.md

---
description: Use the [[= product_name_cloud =]] CLI to manage your I[[= product_name_cloud =]] projects from the command line.
month_change: true
---

# [[= product_name_cloud =]] CLI

The [[= product_name_cloud =]] CLI (`ibexa_cloud`) is a command-line tool for managing your [[= product_name_cloud =]] projects.
It is based on the [Upsun CLI](https://docs.upsun.com/administration/cli.html) and shares the same commands.

## Installation

Follow the installation instructions at [cli.ibexa.cloud](https://cli.ibexa.cloud/).

After installation, authenticate with your [[= product_name_cloud =]] account:

```bash
ibexa_cloud auth:browser-login
```

## Command reference

> For the full list of available commands, see the [Upsun CLI reference](https://developer.upsun.com/cli/reference).

Reviewer: ```suggestion
For the full list of available commands, run `ibexa_cloud list` or see the [Upsun CLI reference](https://developer.upsun.com/cli/reference).
```

Replace `upsun` with `ibexa_cloud` in all examples.

## Examples

### Run a SQL script

To execute a SQL upgrade script on a [[= product_name_cloud =]] environment, pass it to `ibexa_cloud sql`:

=== "MySQL"

    ```bash
    ibexa_cloud sql < vendor/ibexa/installer/upgrade/db/mysql/ibexa-x.x.x-to-x.x.y.sql
    ```

=== "PostgreSQL"

    ```bash
    ibexa_cloud sql < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-x.x.x-to-x.x.y.sql
    ```

### Connect with a SQL client

> To connect to the database using a graphical SQL client, start a SSH tunnel by running the following command in the project directory:

Reviewer 2: "local SQL client", or "any SQL client", or just "SQL client". Doesn't need to be graphical ;)

Author: Fixed in https://github.com/ibexa/documentation-developer/pull/3198/commits/27b242ec488d939f9a8b63a714b992fcc75f75aa , thanks 😄

```bash

> ibexa_cloud tunnel:open

Reviewer 3: Just so we are clear.. This will open a ssh tunnel to all services (db, redis, solr etc), not only to the DB.
Maybe that should be pointed out?

Only execption I know how is varnish if you have that.. No tunnel to varnish will be estabilished

---
description: Use the [[= product_name_cloud =]] CLI to manage your I[[= product_name_cloud =]] projects from the command line.
month_change: true
---

# [[= product_name_cloud =]] CLI

The [[= product_name_cloud =]] CLI (`ibexa_cloud`) is a command-line tool for managing your [[= product_name_cloud =]] projects.

> It is based on the [Upsun CLI](https://docs.upsun.com/administration/cli.html) and shares the same commands.

Reviewer 4: ```suggestion
It's based on the [Upsun CLI](https://docs.upsun.com/administration/cli.html) and shares the same commands.
```
